apps/react/src/features/genres/pages/GenresPage/GenresPage.tsx

```diff
@@ -20,14 +25,21 @@ const GenresPageComponent: FC = () => {
 	}
 
 	return (
-		<>
-			<h1>Genres</h1>
-			{genres.map(genre => <GenreCard key={genre.id} genre={genre} />)}
-		</>
+		<main className={styles.main}>
+			<Box className={styles.main__section}>
+				<GenresFilterForm/>
+				<Box className={styles.main__section__list}>
+					<GenresList genres={genres} />
+				</Box>
+			</Box>
+			<Box className={styles.main__details}>
+				<Outlet />
+			</Box>
```

Comment:
If you don't need to use the Box features, it's better to use a div

```suggestion
			<div className={styles.main__section}>
				<GenresFilterForm/>
				<div className={styles.main__section__list}>
					<GenresList genres={genres} />
				</div>
			</div>
			<div className={styles.main__details}>
				<Outlet />
			</div>
```